cargo-shuttle/src/lib.rs

```diff
@@ -855,18 +855,15 @@ impl Shuttle {
         {
             let dir_entry = dir_entry.context("get directory entry")?;
 
-            if dir_entry.file_type().context("get file type")?.is_dir() {
-                let secrets_path = dir_entry.path().join("Secrets.toml");
-
-                // Make sure to add any `Secrets.toml` files in the subdirectories.
-                if secrets_path.exists() {
-                    let path = secrets_path
-                        .strip_prefix(base_directory)
-                        .context("strip the base of the archive entry")?;
-                    tar.append_path_with_name(secrets_path.clone(), path)?;
-                }
+            let secrets_path = dir_entry.path().join("Secrets.toml");
+
+            // Make sure to add any `Secrets.toml` files.
+            if secrets_path.exists() {
+                tar.append_path_with_name(secrets_path, Path::new("shuttle").join("Secrets.toml"))?;
```

Comment:
Won't this override the same `Secret.toml` the whole time? I guess it's also fine since we are not supporting multiple services yet